packages/web-components/fast-foundation/src/anchored-region/anchored-region.ts

```diff
@@ -878,6 +882,9 @@ export class AnchoredRegion extends FASTElement {
 
         if (!this.initialLayoutComplete) {
             this.initialLayoutComplete = true;
+            this.classList.toggle("loaded", true);
```

Comment:
Is there a reason you're not doing this work in the template itself?

Reply:
That would be better, done.